### DISCUSSION TITLE
Youtube Video 4 Anarchy-Droid (Spanish)

### DISCUSSION THREAD

**Discussion by DanRatia** [amo13/Anarchy-Droid#24] [Show and tell]
If you need, here you have a videotutorial for your tool, in spanish language: https://youtu.be/hdAZ4UGs2Sc

**Comment by amo13** [comment]
Wow, I'm impressed!
Glad you like the application!

**Reply by DanRatia** [reply]
Your program is very good and works correctly.  One suggestion: I would include the bypass for the anti-rollback system, it causes a lot of bricks in novice users.
Reggards

**Reply by amo13** [reply]
Thanks for the idea. But I'm not sure what you mean by anti-rollback system?

**Reply by DanRatia** [reply]
Some devices, like Xiaomi Redmi Note 5 Pro I use on the video of your tool, has this system to prevent you con flash previous version of the software that you have on your device. This prevents to install TWRP on it too.
Read this: https://www.xda-developers.com/xiaomi-anti-rollback-protection-brick-phone/amp/

**Reply by amo13** [reply]
Thank you for the link.
I guess this is an important takeaway:

    How to find current rollback index

    Reboot to fastboot mode
    Enter the following command: fastboot getvar anti
    If the output is blank, then anti-rollback has not yet been enabled. If you get a number in the output, then that’s your current rollback index.

Are you saying that this anti-rollback system is safe to bypass? And if so, how would it be bypassed?

I have never used a Xiaomi device because the unlock procedure looked kind of complicated to me. Therefore, I have no idea of how any of that works on Xiaomi.

**Reply by DanRatia** [reply]
I tell you because I work a lot of with Xiaomi on my channel (i already warned to my public) and I you try to flash twrp without bypass it, you brick (if ARB).
With getvar:
If you got 1, no ARB problem, like my Redmi K30 5G.
If you got 2 o +, ARB problem, like my Redmi Note 5
Other tools like Xiaomi ADB Fastboot Tools have the ARB bypass option, flashing a "dump.img" file by fastboot way.
I have some videos on my channel, but in Spanish language.
